### DISCUSSION TITLE
Time saving feature request for GL entry

### DISCUSSION THREAD

**Discussion by zl3ag** [ledgersmb/LedgerSMB#5860] [Ideas]
In the general journal entry screen, if your firstline  entry is a credit of (say) $500, when you click on the debit field on the next line, it is automatically pre-filled with $500 (as that's most likely what going to go there) with the entire figure pre-selected for 1-key deletion. If you replace the 2nd $500 with $450, then click on the debit field on the 3rd line down, it auto fills with $50.

**Comment by einhverfr** [comment]
The plan in 1.6 is to move the whole gl screen to a dojo widget so this would be quick and easy to add at that time and would provide for a fairly responsive approach.
